test/extended/builds/volumes.go

```diff
@@ -51,7 +49,7 @@ var _ = g.Describe("[sig-builds][Feature:Builds][volumes] build volumes", func()
 			}
 		})
 
-		g.It("should mount given secrets and configmaps into the build pod for source strategy builds [apigroup:image.openshift.io][apigroup:build.openshift.io][apigroup:apps.openshift.io]", func() {
+		g.It("should mount given secrets and configmaps into the build pod for source strategy builds [apigroup:image.openshift.io][apigroup:build.openshift.io]", func() {
```

Comment:
Can't remember if we discussed this before but you may want to rename these tests in the test mapping repo so history carries over and component readiness can compare to past releases. Unless, you feel this is now a new test that should not be compared anymore?

"New" tests may soon be required to have a very high pass rate 99% by component readiness.